### ISSUE TITLE
Train track has a weird behaviour

### ISSUE THREAD

**Issue by astronomersiva** [pythonindia/inpycon2019#96]
<img width="169" alt="Screen Shot 2019-06-01 at 23 09 33" src="https://user-images.githubusercontent.com/7725225/58751865-608ebd80-84c2-11e9-902d-dfcc50604cc9.png">

It renders only after scrolling. It kinda looks funny and needs to be fixed.

**Comment by Sayak9495** [comment]
@astronomersiva Is it happening with whole train and train track? Because In my system everything is rendering at once.

**Comment by astronomersiva** [comment]
Can you try it zoomed out or in higher resolution?

**Comment by Sayak9495** [comment]
Yes when I zoom out after 75% the train disappears. And in my case even after scrolling its not rendering.
![image](https://user-images.githubusercontent.com/16087116/58758698-42fe3a00-853c-11e9-8b7a-f4e52fb2a85d.png)

 @astronomersiva can you please suggest how to reproduce the exact rendering error you are facing? Cause I tried with IE as well and at normal condition it is rendering as expected.

**Comment by astronomersiva** [comment]
![track](https://user-images.githubusercontent.com/7725225/58763212-993b9f00-8575-11e9-8168-a04798219501.gif)

Notice how the track's front portion is missing

**Comment by Sayak9495** [comment]
@astronomersiva I will look into the code and try to fix it asap.
